packages/frontend/src/stores/messagelist.ts

```diff
@@ -309,31 +309,33 @@ class MessageListStore extends Store<MessageListState> {
     loadChat: this.scheduler.lockedQueuedEffect(
       'scroll',
       async () => {
-        const chat = await BackendRemote.rpc.getFullChatById(
+        const firstUnreadMsgIdP = BackendRemote.rpc.getFirstUnreadMessageOfChat(
           this.accountId,
           this.chatId
         )
-        if (chat.id === null) {
-          this.log.debug(
-            'SELECT CHAT chat does not exist, id is null. chatId:',
-            chat.id
-          )
-          return
-        }
-        const messageListItems = await BackendRemote.rpc.getMessageListItems(
+        const messageListItemsP = BackendRemote.rpc.getMessageListItems(
           this.accountId,
           this.chatId,
           false,
           true
         )
+        const chatP = BackendRemote.rpc.getBasicChatInfo(
+          this.accountId,
+          this.chatId
+        )
+        chatP.then(chat => {
+          if (chat.id === null) {
+            this.log.debug(
+              'SELECT CHAT chat does not exist, id is null. chatId:',
+              chat.id
+            )
+          }
+        })
 
-        const firstUnreadMsgId =
-          await BackendRemote.rpc.getFirstUnreadMessageOfChat(
-            this.accountId,
-            this.chatId
-          )
+        const firstUnreadMsgId = await firstUnreadMsgIdP
         if (firstUnreadMsgId !== null) {
-          setTimeout(() => {
+          setTimeout(async () => {
```

Comment:
here a comment would be nice:
"interrupt loadChat here and switch to jumpToMessage which loads chat and messageListItems"

to explain why we return false

Reply:
Perhaps, though adding comments is not a part of this MR.

Reply:
Yes. That's why I approved it. I added it only, since you added comments in other parts of your changes which I appreciate :-)